File: software/examples/java/ExampleSimpleCurrent.java
import com.tinkerforge.BrickletIndustrialAnalogOut;
import com.tinkerforge.IPConnection;

public class ExampleSimpleCurrent {
	private static final String HOST = "localhost";
	private static final int PORT = 4223;
	private static final String UID = "XYZ"; // Change to your UID

	// Note: To make the example code cleaner we do not handle exceptions. Exceptions you
	//       might normally want to catch are described in the documentation
	public static void main(String args[]) throws Exception {
		IPConnection ipcon = new IPConnection(); // Create IP connection
		BrickletIndustrialAnalogOut ao = new BrickletIndustrialAnalogOut(UID, ipcon); // Create device object

		ipcon.connect(HOST, PORT); // Connect to brickd
		// Don't use device before ipcon is connected

		// Set output current to 4.5mA
		ao.setCurrent(4500);
		ao.enable();

		System.out.println("Press key to exit"); System.in.read();
		ipcon.disconnect();
	}
}

File: software/examples/vbnet/ExampleSimpleCurrent.vb
Imports System
Imports Tinkerforge

Module ExampleSimpleCurrent
    Const HOST As String = "localhost"
    Const PORT As Integer = 4223
    Const UID As String = "XYZ" ' Change XYZ to the UID of your Industrial Analog Out Bricklet

    Sub Main()
        Dim ipcon As New IPConnection() ' Create IP connection
        Dim iao As New BrickletIndustrialAnalogOut(UID, ipcon) ' Create device object

        ipcon.Connect(HOST, PORT) ' Connect to brickd
        ' Don't use device before ipcon is connected

        ' Set output current to 4.5mA
        iao.SetCurrent(4500)
        iao.Enable()

        Console.WriteLine("Press key to exit")
        Console.ReadLine()
        iao.Disable()
        ipcon.Disconnect()
    End Sub
End Module
